core/names.ts

```diff
@@ -32,8 +32,7 @@ import {Workspace} from './workspace.js';
  * @alias Blockly.Names
  */
 export class Names {
-  static NameType: AnyDuringMigration;
-  static DEVELOPER_VARIABLE_TYPE: AnyDuringMigration;
+  static DEVELOPER_VARIABLE_TYPE: NameType;
```

Comment:
```suggestion
  static DEVELOPER_VARIABLE_TYPE: NameType.DEVELOPER_VARIABLE;
```

Reply:
Ohhh woops gotcha.

> Oh, I realise I can probably write a fancier type spec here that declares that .DEVELOPER_VARIABLE_TYPE can only be one particular value of NameType, but I'm not sure it's worth being that specific.

That would be fun, but yeah I don't think it's worth being that specific either lol.